### ISSUE TITLE
Delete file bug on OTG storage

### ISSUE THREAD

**Issue by cosmin86** [SimpleMobileTools/Simple-File-Manager#602]
Versions newer than 6.11.3 are unable to delete files on removable storage (OTG) - something about a Java error. I'm using Android 11 RKQ1.200826.002 MIUI 12.1.1.0 (RFJEUXM) on a Xiaomi Mi 9T smartphone.

**Comment by tibbi** [comment]
tell the exact error message please

**Comment by cosmin86** [comment]
I documented the behavior of version 6.12.4 Pro (from F-Droid) with the following examples on a 64Gb exfat USB stick (all the stored data for the app was cleared after the update, permission to access the OTG drive was granted):

1. I tried to create a folder on the USB drive. Nothing happens on "Ok" command (first screenshot).
2. I deleted the first test file. The "Error: java.util.NoSuchElementException" flashed on the screen (second screenshot). After a refresh the deleted file reappeared.
3. I tried to move the screenshots on the USB drive. The "Error: java.io.FileNotFoundException: {file location}: open failed: ENOENT (No such file or directory)" (third screenshot) followed by "Error: java.lang.NullPointerException" (fourth screenshot).

![Screenshot_2022-06-24-11-43-23-508_com simplemobiletools filemanager pro](https://user-images.githubusercontent.com/108103856/175504045-3580e38b-4315-428b-ae59-589792758be4.jpg)
![Screenshot_2022-06-24-11-43-37-656_com simplemobiletools filemanager pro](https://user-images.githubusercontent.com/108103856/175504062-1aeb1965-9065-4500-98b5-5fb08154bddd.jpg)
![Screenshot_2022-06-24-11-45-12-498_com simplemobiletools filemanager pro2](https://user-images.githubusercontent.com/108103856/175504076-390ac8c5-b8f8-4d66-b45b-66f5496f91a6.jpg)
![Screenshot_2022-06-24-11-45-16-415_com simplemobiletools filemanager pro2](https://user-images.githubusercontent.com/108103856/175504087-9544aafc-e96f-4dcc-bc77-2b31ac36a111.jpg)
.
